woocommerce/woocommerce
# Revert "Add To Cart Form: Add `Stepper` layout"

This reverts commit 0712c6037b4e8a868c1e66ff18fc7046e606a23f https://github.com/woocommerce/woocommerce/pull/47664

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR reverts https://github.com/woocommerce/woocommerce/pull/47664. We noticed that there are some layout issues with the payments plugin: we decided to revert this PR and try to iterate the CSS in the following weeks: https://github.com/woocommerce/woocommerce/issues/48240



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable TT4.
2. Edit the Single Product Template.
3. Ensure that you have the Add To Cart Form block added.
4. Click on it.
5. Open the block settings.
6. Ensure that the `Quantity Selector` option **isn't visible**



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

This PR is just a revert.

</details>
